### PR TITLE
RT 4753 - Display the logs in console

### PR DESCRIPTION
**Pull request by slashd0t**
Display the logs in console and also write the logs in development.log file

### CODE REVIEW

**Comment by jaygel179** [comment]
can you please fix the PR? why there are other commits that is not related to the fix?

File: lib/stackify/logger_client.rb [hunk 1]
```diff
@@ -12,7 +12,10 @@ def initialize
       end
     end
 
-    def log level, msg, call_trace
+    def log num_level, level, msg, call_trace
```
Comment (darinhoward) [inline lib/stackify/logger_client.rb:15]: add documentation to this function - include what each parameter is used for

File: lib/stackify/authorization/authorizable.rb [hunk 2]
```diff
@@ -11,7 +11,9 @@ module ClassMethods
     @@auth_client = nil
 
     def authorize attempts=3
-      Stackify::EnvDetails.instance.set_rails_info
+      if Gem::Version.new(RUBY_VERSION) <= Gem::Version.new('2.0')
```
Comment (darinhoward) [inline lib/stackify/authorization/authorizable.rb:14]: comment in code why we are adding this condition on the set_rails_info call

**Comment by Envek** [comment]
I tried this pull request and it causes duplication of some log entries if logging is already set up directly to stdout (that often the case in containerized production environments).

Most probably you tried to replicate [internal Rails log splitter](https://github.com/rails/rails/blob/492b9b413d3aa3a9bf3bf986ae25c7f90ec1ab62/railties/lib/rails/commands/server/server_command.rb#L77-L87) for [development](https://github.com/rails/rails/blob/492b9b413d3aa3a9bf3bf986ae25c7f90ec1ab62/railties/lib/rails/commands/server/server_command.rb#L124-L125) (which has check to avoid stdout duplication), but if so then your approach is incorrect.

---

`config/environments/development.rb`:
```ruby
  config.logger = ActiveSupport::Logger.new(STDOUT)
  config.log_level = :debug
```
`config/initializers/stackify.rb`:
```ruby
Stackify.setup do |config|
  config.api_key = 'REDACTED'
  config.app_name = "REDACTED"
  config.env = :development
end
```
Rails 5.2.3

Result (`Started GET` line is duplicated, others are not):
```
rails_1      | Started GET "/admin/top/index" for 172.22.0.1 at 2020-06-01 17:12:07 +0000 by …
rails_1      | Started GET "/admin/top/index" for 172.22.0.1 at 2020-06-01 17:12:07 +0000 by …
rails_1      | Processing by Admin::TopController#index as HTML
rails_1      |    (0.3ms)  SET NAMES …
rails_1      |   ↳ app/services/admin_user_service.rb:126
rails_1      |   Model Load (0.4ms)  SELECT …
rails_1      |   ↳ app/services/admin_user_service.rb:126
rails_1      |    (0.4ms)  SELECT …
rails_1      |   ↳ app/services/admin_user_service.rb:130
rails_1      |   Model Load (0.7ms)  SELECT  …
rails_1      |   ↳ app/services/access_control_service.rb:9
```

**Pull request merged**